internal/handlers/users_v1_handler.go

```diff
@@ -7,6 +7,7 @@ import (
 
 	"github.com/redhatinsights/mbop/internal/config"
 	"github.com/redhatinsights/mbop/internal/service/ocm"
+	"github.com/redhatinsights/mbop/internal/service/ocm/keycloak"
```

Comment:
I feel like this shouldn't be coupled to/namespaced under OCM since the Keycloak user service isn't directly related to OCM?